### DISCUSSION TITLE
.js should be interpreted as .jsx on React projects, right?

### DISCUSSION THREAD

**Discussion by marcosbacon** [brackets-cont/brackets#169] [Q&A] [answered]
I even clicked on "Set jsx as default for this file" to every .js file in my React app, mas it will reset every time I reopen Brackets. Any ideas?

![image](https://user-images.githubusercontent.com/5806785/159168581-6a6415f4-d806-4924-8908-217edd89af99.png)

**Comment by kirtisingh3008** [comment, marked as answer]
I tried doing the same, but it works fine for me like go to the file and rename it and after that, it will by default change the file type to jsx.
And even after you close the folder and reopen it would be jsx only. I have attached a screenshot for the same.
![Screenshot (638)_LI](https://user-images.githubusercontent.com/70806481/159216473-198d1560-6d46-4035-8828-6c1824dce5bb.jpg)
I hope this answers your question. Thank you!

**Reply by marcosbacon** [reply]
Thank you! I guess the compilers will figure out by itself in the end what to do with the files, so in this case jsx and js are interchangeable.
